signac/contrib/job.py

```diff
@@ -219,10 +219,9 @@ def reset_statepoint(self, new_statepoint):
         # Update this instance
         self.statepoint._data = dst.statepoint._data
         self._id = dst._id
-        self._wd = dst._wd
-        self._fn_doc = dst._fn_doc
+        self._wd = None
         self._document = None
-        self._data = None
+        self._stores = None
```

Comment:
This is a bug left in from long ago. The `_data` attribute doesn't exist, it's called `_stores`.